### ISSUE TITLE
CortanaVoiceCommand not working

### ISSUE THREAD

**Issue by ArnonEilat** [Microsoft/Windows-universal-samples#434]
i'm trying to run the [CortanaVoiceCommand](https://github.com/Microsoft/Windows-universal-samples/tree/master/Samples/CortanaVoiceCommand), both the JS version and the C# version.

The build is going well and i can see the AdventureWorks window. But when i write(or say) to Cortana:
- Adventure Works, show trip to London
- Adventure Works, when is my trip to London
- Adventure Works, cancel trip to London
- London
- Melbourne
  Nothing happen....

See screen shot:
![screenshot](https://cloud.githubusercontent.com/assets/1027943/17140945/1256243a-5353-11e6-8bed-1d024cb1cd17.png)

In the "cortana powered apps" i can see the AdventureWorks.
Screen shot:
![screen shot1](https://cloud.githubusercontent.com/assets/1027943/17141127/a4f5906e-5353-11e6-95c3-e45bbb012fd6.png)

I'm running(and compiling) on virtual machines with Win 10 Preview (14.14393) from [here](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/)

Is the problem in my Cortana?
Is the code missing something?

**Comment by anpilley** [comment]
Off the bat, the first thing I'd mention is that the text entry mechanism is generally fairly unforgiving of punctuation at the moment. When you're typing "Adventure works, Show trip to London", the comma can cause some issues that prevent the phrase from being recognized. That typically manifests in the behavior you're seeing above. You'll notice that the comma is not present in AdventureWorksCommands.xml. We're aware of that limitation and are investigating ways to improve it for the future.

That said, you do mention having trouble with voice as well. It can take a minute or two for the commands to fully register, but by the time they're showing up in the help output you show above, it should be available for voice, so that's concerning. Have the commands never worked, even several minutes after the app's first run?

**Comment by ArnonEilat** [comment]
Thanks for the response

Unfortunately the commands never worked.
I've wait an hour and try a lot of permutations but none of them works...

I don't think its the punctuation. I guess it's about the my MS account or the Cortana settings: I'm living in Israel, Cortana ain't working here by default.
To make here working you should [change some settings](https://www.google.com/#newwindow=1&q=use+cortana+outside+us+windows+10). Although in my version of windows 10 its works by default its looks like Cortana not work as in the [demos](https://www.google.com/#q=cortana+demo+windows+10&tbm=vid).

Is Cortana dumping error logs somewhere?

**Comment by anpilley** [comment]
Hm. Cortana is heavily dependent on regional data to function correctly, so being in Isreal, and setting Cortana to En-US could affect it, and neither Isreal nor Hebrew are supported for Cortana right now. Getting the language settings combo right is tricky, but usually results in Cortana not working at all, and refusing to turn on, but yours at least appears to be starting up okay.

Are you able to get things like "Wake me up at 8am" to set an alarm correctly?

**Comment by ArnonEilat** [comment]
I'm able to set alarm.

When i'm type: "Wake me up at 8am" - i see this:
![1](https://cloud.githubusercontent.com/assets/1027943/17217731/0b540cb4-54ee-11e6-91d0-c3e17389d69e.png)

Hitting Enter set an alarm correctly:
![2](https://cloud.githubusercontent.com/assets/1027943/17217810/5e9de124-54ee-11e6-852a-f7b622a15895.png)

Is it OK?

**Comment by anpilley** [comment]
Hm, then unfortunately, I'm not sure what could be causing the VCD for Adventure Works to not function correctly. I'll pass this thread across to the feature team and see what they can suggest.

**Comment by Lazzarus666** [comment]
Having the same issue. 
I found some references to region problems on stack overflow and tried switching to en-US but no change. My original region was en-GB

**Comment by anpilley** [comment]
I built a system using he-IL, and migrated it to en-US (change the default language, install the en-US language pack, install the speech pack, change the region to United States) and I was able to get the commands to register and activate okay, although I had to reboot an additional time after switching Cortana across to be English - US as well. It's not a simple process to change languages, however.

I can't reproduce this problem here. If you run into it again, filing a new feedback issue under the "Cortana and Search" -> "Speech Recognition and Audio Issues", and then providing a link to the feedback item should allow for deeper investigation.
